src/PopEngine/index.ts

```diff
@@ -156,6 +156,7 @@ export class PopEngine {
 
       if (isAlreadyShowing && !!container) {
         // if pop is already showing for group, reuse
+        this._fireEvent('Content_Swap', oldPop);
```

Comment:
Can you fire this after the element removal? Otherwise when angular removes reference, the line below fails to remove an element that does not exist.